activejob/test/support/integration/adapters/sneakers.rb

```diff
@@ -84,7 +84,7 @@ def bunny_publisher
     end
 
     def bunny_queue
-      @queue ||= bunny_publisher.exchange.channel.queue "integration_tests", durable: true
+      @queue ||= bunny_publisher.send(:exchange).channel.queue "integration_tests", durable: true
```

Comment:
Looks like `exchange` has been removed from the public API. Do you think there's any chance that we can implement this without reaching into the object? If not, maybe we can persuade the sneakers maintainer to consider making it public again...

/cc @jondot